devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -270,6 +312,63 @@ def normalizeLevel(level) {
 	level == 99 ? 100 : level
 }
 
+def getAeotecNanoDimmerDefaults() {
+	[
+		min: 0,
+		max: 99
+	]
+}
+
+def getConfigurationCommands() {
```

Comment:
I would name this `getAeotecNanoDimmerConfigurationCommands` as this is specific to this device.

Reply:
sure, fixed